### ISSUE TITLE
Place rooms in configured community

### ISSUE THREAD

**Issue by ChrisLane** [Sorunome/mx-puppet-bridge#49]
I have a lot of rooms on my homeserver due to all of the bridges that I'm running and my activity within them.
I think it would be useful for rooms to be configured to be placed in specific communities for each bridge.

For example, I could create a community `+discord:example.com` and then configure all rooms created by the `mx-puppet-discord` bridge to be placed in that community.

**Comment by Sorunome** [comment]
If you enable group sync it'll make a matrix community/group per discord server/guild and sort based in that.

If you want to sort more, you might want to look into room tags.

Am 30. Juni 2020 12:38:37 MESZ schrieb Chris Lane <notifications@github.com>:

**Comment by ChrisLane** [comment]
That's enough for me but I do think some people might want to format their room names as ":guild | :name" and then have them all in a "Discord" community for example.

**Comment by Sorunome** [comment]
well, that is what name patterns exist for, you can configure there how the name should look like.

**Comment by ChrisLane** [comment]
Yes but that doesn't cover putting them in a custom configurable community as well does it?

**Comment by Sorunome** [comment]
It sounds like you actually want a way to get admin in a community, so that you can manually configure it to your looks

**Comment by robintown** [comment]
Another user has requested this feature: https://gitlab.com/robintown/mx-puppet-groupme/-/issues/4

**Comment by corrupt3k** [comment]
I am also interested in this. I am using matrix for aggregating all of my personal chat services together. I have a mockup.
![image](https://user-images.githubusercontent.com/20589688/105901458-a45d0180-5fda-11eb-9fa3-2e16a532372d.png)

**Comment by Sorunome** [comment]
Yes, you can add a community to your registration file which all ghosts will be added to, with the `group_id` parameter

**Comment by minecraftchest1** [comment]
What about using spaces?

**Comment by dsonck92** [comment]
Spaces support is already being worked on for this bridge, and the Discord project being one of the first to receive support for it allowing grouping several discord guilds under a common Discord space. Community support will be dropped somewhere down the road considering spaces are superseding it.
